kin-devplatform-sdk/src/main/java/kin/devplatform/data/blockchain/BlockchainSourceImpl.java

```diff
@@ -422,6 +423,13 @@ public void updateActiveAccount(IKinClient kinClient, int accountIndex) throws B
 		getBalance(null);
 	}
 
+	@Override
+	public void deleteAccount(int accountIndex) throws DeleteAccountException {
+		if (local.getAccountIndex() != accountIndex) {
```

Comment:
worth checking also for the case where the imported index is smaller than the current active index (i.e. the imported account was there already) in this case no need for delete. o.w. we will mess up the indexes :frowning_face: